### PR TITLE
Add gradient accumulation and gradient clipping to text benchmark script

### PR DESCRIPTION
**Pull request by zechengz**
Add gradient accumulation and gradient clipping to text benchmark script
TODO: add warmup

### CODE REVIEW

**Review by vid-koci — APPROVED** [review]

**Comment by zechengz** [comment]
Close this as it's not necessary.
